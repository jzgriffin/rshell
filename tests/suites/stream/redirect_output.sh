grep -i foo redirect_input.out > redirect_output_1.tmp; cat redirect_output_1.tmp
echo hello world >redirect_output_2.tmp; cat redirect_output_2.tmp
rm redirect_output_1.tmp redirect_output_2.tmp
