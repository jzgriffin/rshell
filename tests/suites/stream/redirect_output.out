FOO
foo
foo
hello world
